### PR TITLE
WIP: Add benchmarks page

### PR DESCRIPTION
**Pull request by RFMVasconcelos**
This placeholder page has been requested on the on-prem WG.

It aims at adding benchmarks around Networking and Storage solutions to use with Kubeflow. See [meeting logs](https://docs.google.com/document/d/1Qs94d9RPxqiZJnTRvL9iHZrx_jnf59HyJFnbyi1AYjQ/edit).

### CODE REVIEW

**Comment by Jeffwan** [comment]
@xyhuang

**Comment by Jeffwan** [comment]
is there a plan to add more contents in the website?

**Comment by rmgogogo** [comment]
@joeliedtke

**Comment by rmgogogo** [comment]
I'm wondering whether we can provide benchmark result together in one PR. I didn't attend the WG meetings and may missed few context.

Is it for certain component's network/storage benchmark? E.x. run pipeline and benchmark it? Storage part seems clear to me. I'm not sure about the network part, it's still mainly the storage I/O, right?

File: content/en/docs/other-guides/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
++++
+title = "Benchmarks"
+description = "Benchmarks on networking and storage solutions to use with Kubeflow"
+weight = 105
+                    
++++
+
+This docs page is a placeholder. 
+ 
+Soon it will serve as reference for users to choose networking and storage solutions.
```
[the remark this answers is not in the document]
Comment (RFMVasconcelos) [reply content/en/docs/other-guides/benchmarks.md]: Added the first 2 suggestions. Is there a neet way to show the WIP, or just add that as text?

**Comment by joeliedtke** [comment]
Would you please add some more context around why it is important to add a placeholder page? It seems like it would be better to hold off on publishing this page until we have some benchmarks available.

**Comment by RFMVasconcelos** [comment]
Hi @joeliedtke, 

This was a request by the on-prem WG, but I guess we can wait until we have first benchmarks to merge. I may change this PR to WIP and ask people to commit the content here. WDYT?

**Comment by jlewi** [comment]
+1 to @joeliedtke  lets not create empty pages.  Before we start documenting benchmarks lets make sure we have a consistent way for maintaining and updating them. If its a one time thing then a blog post is probably better suited.

Also there is no official WG.

All WGs are listed here
https://github.com/kubeflow/community/tree/master/wgs

**Comment by RFMVasconcelos** [comment]
Oh, I see. Just trying to help 😄

I'll change this PR to WIP and await for relevant additions from the community. 

Regarding WG, I guess we need to make that call official, either WG or SIG. I guess I can try to accelerate that process...

**Comment by catblade** [comment]
I had requested this be created so we could start collaborating to come up with benchmarks and expected behavior for various system types.  It is hard to know if Kubeflow and it's underlying pieces are configured correctly without these numbers.  The hope was to get community involvement so we could work together to come up with a cohesive set.  We have people at my current company that would really like to collaborate on it.  Having a place to work on this is very much desired.

**Comment by jlewi** [comment]
@catblade Can we use an issue or mailing list thread to coordinate collaboration on a benchmark. Lets not create user facing docs until we have something thats ready to share with users.

I'm going to go ahead and close this PR.

**Pull request closed without merging**